### PR TITLE
TOC in admin manual - maintenance

### PR DESCRIPTION
**Pull request by mmattel**
Follow up to #872 (TOC in admin manual - installation)

Some text fixes included.
Corrected the physical location of `upgrading/` into `maintenance/upgrading/` (incl nav fix)
It is shown in maintenance but the physical location was different.

### CODE REVIEW

File: modules/admin_manual/pages/maintenance/enable_maintenance.adoc [hunk 1]
```diff
@@ -8,18 +8,17 @@ Please see xref:configuration/server/occ_command.adoc[Using the occ Command] to
...
-....
+[source,console]
+----
+sudo -u www-data php occ maintenance:mode --on
+----
 
-You may also put your server into this mode by editing
-config/config.php. Change `"maintenance" => false` to
-`"maintenance" => true`:
+You may also put your server into this mode by editing config/config.php. +
```
Comment (settermjd) [inline modules/admin_manual/pages/maintenance/enable_maintenance.adoc]: Can we avoid this practice, pls?

Comment (mmattel) [reply modules/admin_manual/pages/maintenance/enable_maintenance.adoc]: If you look to the rendered output before and after, you see the difference and it is much better readable now.

File: modules/admin_manual/pages/maintenance/manual_upgrade.adoc [hunk 2]
```diff
@@ -207,8 +221,7 @@ With maintenance mode disabled, login and:
 * Check that the version number reflects the new installation. It’s
 visible at the bottom of your Admin page.
 * Check that your other settings are correct.
-* Go to the Apps page and review the core apps to make sure the right
-ones are enabled.
+* Go to the Apps page and review the core apps to make sure the right ones are enabled.
```
Comment (settermjd) [inline modules/admin_manual/pages/maintenance/manual_upgrade.adoc]: How about using the `:experimental:` functionality here, to make the navigation that much clearer?

File: modules/admin_manual/pages/maintenance/upgrading/marketplace_apps.adoc [hunk 3]
```diff
@@ -1,18 +1,19 @@
...
 == Single-Server Environment
 
 To upgrade Marketplace applications when running ownCloud in a single server environment, you can use use xref:configuration/server/occ_command.adoc#apps-commands[the Market app], specifically by running `market:upgrade`.
 This will install new versions of your installed apps if updates are available in the marketplace.
 
 NOTE: The user running the update command, which will likely be your webserver user, needs write permission for the `/apps` folder. If they don’t have write permission, the command may report that the update was successful, however it may silently fail.
 
-[[clusteredmulti-server-environment]]
-== Clustered/Multi-Server Environment
+== Clustered / Multi-Server Environment
```
Comment (settermjd) [inline modules/admin_manual/pages/maintenance/upgrading/marketplace_apps.adoc]: Nice change; making it that much simpler to read.

File: modules/admin_manual/pages/maintenance/upgrading/marketplace_apps.adoc [hunk 4]
```diff
@@ -21,12 +22,9 @@ the cluster individually. There are several ways to do this. But here is
 a concise approach:
 
 1.  Download the latest server release (whether
-https://download.owncloud.org/community/owncloud-10.0.4.tar.bz2[the
-tarball] or
-https://download.owncloud.org/community/owncloud-10.0.4.zip[the zip
-archive]).
+https://download.owncloud.org/community/owncloud-10.0.4.tar.bz2[the tarball] or
```
Comment (settermjd) [inline modules/admin_manual/pages/maintenance/upgrading/marketplace_apps.adoc]: How about storing the links as page-level attributes?

**Pull request merged**